command/pr.go

```diff
@@ -471,41 +482,146 @@ func prMerge(cmd *cobra.Command, args []string) error {
 		}
 	}
 
-	if pr.State == "MERGED" {
+	if pr.Mergeable == "CONFLICTING" {
+		err := fmt.Errorf("%s Pull request #%d has conflicts and isn't mergeable ", utils.Red("!"), pr.Number)
+		return err
+	} else if pr.Mergeable == "UNKNOWN" {
+		err := fmt.Errorf("%s Pull request #%d can't be merged right now; try again in a few seconds", utils.Red("!"), pr.Number)
+		return err
+	} else if pr.State == "MERGED" {
```

Comment:
I think we should check this state before any checks around the `.Mergeable` field. If the PR is already merged, its `.Mergeable` field might report "UNKOWN" or "CONFLICTING", in which case the current logic will trigger one of the error messages above, which would be misleading to the user.